.clang-format

```
@@ -0,0 +1,41 @@
+Language: Cpp
+AccessModifierOffset: 0
+AlignConsecutiveMacros: true
+BreakBeforeBraces: Linux
+IndentWidth: 2
+ConstructorInitializerIndentWidth: 2
```

It seems that the line with specification is missing here compared to the earlier specs. I suggest setting it to "0" to disable automatic line wrapping because no matter how you specify those, automatic wrapping will eventually give you undesired effect. Too long lines of code can be addressed in PR reviews easily.
```suggestion
ColumnLimit: 0
ConstructorInitializerIndentWidth: 2
```